addon/utils/cp/form-validation-class.js

```diff
@@ -0,0 +1,15 @@
+import { computed } from '@ember/object';
+
+export default function formValidationClass() {
+  return computed('validationType', function() {
+    let validationType = this.get('validationType');
+    switch (validationType) {
+      case 'error':
+        return 'is-invalid';
+      case 'success':
+        return 'is-valid';
+      case 'warning':
+        return 'is-warning'; // not officially supported in BS4 :(
+    }
```

Comment:
We may want to add an assertion as default case to ensure that `validationType` is supported.

Reply:
But an empty value (`null`) is valid, in which case no class should be applied.

Reply:
Missed that one.